config/crd/kustomization.yaml

```diff
@@ -3,7 +3,7 @@
 # It should be run by config/default
 # Labels to add to all resources and selectors.
 commonLabels:
-  cluster.x-k8s.io/v1beta1: v1beta1
+  cluster.x-k8s.io/v1beta1: v1beta1_v1beta2
```

Comment:
Should this be `v1beta1`?

Reply:
since our latest version is v1beta2, we have to set it as v1beta1_v1beta2. This is same in other providers as well